test/players/mpd_test.rb

```diff
@@ -1,13 +1,13 @@
-require "test_helper"
-require "ruby-mpd"
-require "anyplayer/players/mpd"
+require 'test_helper'
+require 'ruby-mpd'
+require 'anyplayer/players/mpd'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.